### DISCUSSION TITLE
whats the difference between the-psf.pypi.file_downloads and the-psf.pypi.downloads?

### DISCUSSION THREAD

**Discussion by edenlightning** [psincraian/pepy#287] [General]
I see that you are using the-psf.pypi.file_downloads (as mentioned in https://packaging.python.org/guides/analyzing-pypi-package-downloads/), but other tools such as  https://pypistats.org/ are using the-psf.pypi.downloads.

Whats the logic there? If a package contains multiple files, wouldn't it make more sense to count once?

**Comment by psincraian** [comment]
Hi @edenlightning,

According to  @ewdurbin:

> the original implementation of linehaul that feeds the explicitly partitioned 'downloads' table started having some open file descriptor issues starting on the 4th...
> 
> newer `file_downloads` tables which are populated from cloud functions are stable, so the loss is recoverable

[Source](https://twitter.com/EWDurbin/status/1347251944935583745?s=20)

**Reply by edenlightning** [reply]
thanks!
